app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/feature/AppTpPrivacyFeaturePlugin.kt

```diff
@@ -16,21 +16,31 @@
 
 package com.duckduckgo.mobile.android.vpn.feature
 
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.core.content.edit
 import com.duckduckgo.di.DaggerSet
 import com.duckduckgo.di.scopes.AppScope
 import com.duckduckgo.privacy.config.api.PrivacyFeaturePlugin
 import com.squareup.anvil.annotations.ContributesMultibinding
 import com.squareup.moshi.JsonAdapter
 import com.squareup.moshi.Moshi
 import javax.inject.Inject
+import logcat.logcat
+
+private const val APPTP_FEATURE_SIGNATURE_KEY = "apptp_feature_signature"
```

Comment:
nit: we're mixing `companion object` with high level const. I don't have strong preferences, but in this file maybe we should pick one so that all const are grouped in the same place (?)

Reply:
Good point - fixed!